### DISCUSSION TITLE
How to identify a task?

### DISCUSSION THREAD

**Discussion by JesseAtSZ** [tokio-rs/tokio#3802] [Q&A] [answered]
How to give a name to a task? For example, in async-std, task can be identified by task:: builder:: new(). name(), If there is no similar method, is there another method to identify a task?

**Comment by Darksonn** [comment, marked as answer]
We don't have the ability to give a name to a task in Tokio itself, but you can attach all sorts of information to a task if you use the tracing library. You could also use a [task local](https://docs.rs/tokio/1/tokio/macro.task_local.html) for storing information about a task.
